FMS/Helpers/FacilityDetailDtoScalar.cs

```diff
@@ -1,5 +1,8 @@
 using ClosedXML.Attributes;
+using DocumentFormat.OpenXml.Office2016.Drawing.ChartDrawing;
```

Comment:
Clean up usings.

Reply:
Done. For some reason, they did not show up "dimmed" as they normally do. I need to change my IDE settings